packages/jsxstyle/preact/jsxstyle-preact.tsx

```diff
@@ -0,0 +1,67 @@
+import * as preact from 'preact';
+import { getStyleCache, componentStyles } from 'jsxstyle-utils';
+import { StyleCache } from 'jsxstyle-utils/src/getStyleCache';
+
+import { CSSProperties } from '../cssproperties';
+
+export const cache: StyleCache = getStyleCache();
+
+export interface ComponentPropProps {
+  className?: any;
+  style?: any;
+  [key: string]: any;
+}
+
+export type ComponentProp =
+  | keyof JSX.IntrinsicElements
+  | preact.AnyComponent<any, any>
+  | any;
+
+export interface JsxstyleProps extends CSSProperties {
+  className?: string;
+  component?: keyof JSX.IntrinsicElements | React.ComponentClass | React.SFC;
```

Comment:
Does this work? Using `React` types in `preact`?

Reply:
Nope this is an error. I pasted a bit too much.